index.js

```diff
@@ -165,20 +165,18 @@ let Fixtures = {
                 throw new Error('Max number of login attempts exceeded!');
             }
 
-            return this._adminLogin().then(() => {
-                return this.create(models, options);
-            });
+            return this._adminLogin().then(() => this.create(models, options));
         }
 
-        if (! _.isArray(models)) {
+        if (!_.isArray(models)) {
             models = [models];
         }
 
         // reset `_sessionAttempt`
         this._sessionAttempt = 0;
 
         let url = utils.constructUrl(VERSION, 'bulk');
-        let params = {headers: this._headers};
+        let params = { headers: this._headers };
```

Comment:
I'll be removing the rule that requires this from the sugarcrm/javascript repo later tonight per what you said and Joao's [review summary from before](https://github.com/sugarcrm/thorn/pull/138#pullrequestreview-23100226).